ci-operator/config/openshift/cluster-baremetal-operator/openshift-cluster-baremetal-operator-master.yaml

```diff
@@ -61,6 +61,12 @@ tests:
   commands: make generate-check
   container:
     from: src
+- as: e2e-metal-ipi
+  steps:
+    cluster_profile: packet
+    workflow: baremetalds-e2e
+    env:
+      ENABLE_CBO_TEST: "true"
```

Comment:
We do not support yet this paradigm (and this could require an enhancement to the baremetalds devscript setup step). Instead I'd suggest to follow an approach similar to this one: https://github.com/openshift/release/blob/7412b852184870ea7869eb5ce5ca765c769c244a/ci-operator/config/openshift-metal3/dev-scripts/openshift-metal3-dev-scripts-master.yaml#L26-L34.
Since this a temporary measure, it could be worth to avoid defining the conf step in the ci step registry (since it won't be used by other repos) but directly inline in the ci-operator config file (for more info on this, see the steps registry [doc section](https://ci-operator-configresolver-ui-ci.apps.ci.l2s4.p1.openshiftapps.com/help#step-from-image))